### ISSUE TITLE
Integration Issues Caused by Aging apollo-client Dependency

### ISSUE THREAD

**Issue by Phylodome** [apollographql/reason-apollo#204]
Currently running into dependency issues (basically [this](https://github.com/Astrocoders/reason-apollo-hooks/issues/17)) when attempting to use a  package (`reason-apollo-hooks`) that adds hooks functionality to `reason-apollo`.

Is there a timeline for either the formal inclusion of hooks functionality within this package (conversation seems to have stalled [here](https://github.com/apollographql/reason-apollo/issues/192)), or at the very least for the package to be brought up to date with respect to other deps that would unblock the community to continue developing apace changes to `apollo-client` and `react-apollo`?

As mentioned in https://github.com/apollographql/reason-apollo/issues/202, it's likely this upgrade is non-trivial. But it would be useful to know whether it's in the works, or if a PR would be welcome if someone were to put in the required effort.

**Issue Labels**

- [x] has-reproduction
- [ ] feature
- [ ] docs
- [x] blocking
- [ ] good first issue

**Comment by Yakimych** [comment]
@margaretkrutikova has a PR with those updates in her fork: https://github.com/MargaretKrutikova/reason-apollo/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

It does resolve the `getCurrentResult` crash - would be nice if this could get reviewed and eventually merged into the main repo if possible.

**Comment by Gregoirevda** [comment]
@Phylodome thanks for raising up this issue. reason-apollo@0.17.0 has the upgraded dependencies

**Comment by Phylodome** [comment]
Wonderful. Thank you!
